app/user/signup.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+
+export default function Signup({ onSignup }) {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  return (
+    <>
+      <h1>Signup</h1>
+      <form autoComplete="off">
+        {" "}
+        <div>
+          {" "}
+          <label>Username: </label>{" "}
+          <input
+            id="username"
+            type="text"
+            value={username}
+            onChange={(e) => {
+              setUsername(e.currentTarget.value);
+            }}
+          ></input>{" "}
+        </div>{" "}
+        <br />{" "}
+        <div>
+          {" "}
+          <label>Password: </label>{" "}
+          <input
+            type="password"
+            id="password"
```

Comment:
```suggestion
            id="password"
            type="password"
```
Changed the order to be the same as for the `username` input field